tests/modules/biobambam/bammarkduplicates2/nextflow.config

```diff
@@ -0,0 +1,9 @@
+process {
+
+    publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
+
+}
+
+conda {
+    createOptions = "-c conda-forge"
+}
```

Comment:
```suggestion
```

Reply:
Assume this is ok to remove?

Reply:
It's required in case `conda-forge` is not in the predefined list of channels. I'm using a bare miniconda that has no channel defined, so had to add that option

Reply:
You should be able to set channel priority in `~/.condarc` as shown below:

```
$ cat ~/.condarc
channels:
  - conda-forge
  - bioconda
  - defaults
```

I use miniconda too and this works well. Also means we don't have to add any custom configuration to the modules to make them more portable in general.

Reply:
Yes, I know how to set up the channels, but I didn't want to do it because I'm a bit pedantic when it comes to dependency requirements :) Anyway, I've removed this commit